appeals/web/src/server/lib/mappers/appeal/submappers/agent.mapper.js

```diff
@@ -0,0 +1,46 @@
+import { permissionNames } from '#environment/permissions.js';
+import { formatServiceUserAsHtmlList } from '#lib/service-user-formatter.js';
+import { mapActionComponent } from '../../component-permissions.mapper.js';
+
+/**
+ * @typedef {import('#appeals/appeal-details/appeal-details.types.js').WebAppeal} WebAppeal
```

Comment:
Considering these types are each only used once, they can probably be imported directly in the type definition below rather than being separately imported and aliased (this seems like a remnant from when these were all mapped in a single file)

Reply:
I can't remember what the state was back then you made this comment but they're now used twice 🤷‍♀️